src/math/definitions.lisp

```diff
@@ -64,3 +64,21 @@
   (:documentation "Perform element-wise negation on a quaternion"))
 
 (defgeneric lerp (x t1 t2 val1 val2))
+
+(defgeneric absp (poly)
+  (:documentation "Perform element-wise absolute value on a polynomial"))
+
+(defgeneric negp (poly)
+  (:documentation "Perform element-wise negation on a polynomial"))
+
+(defgeneric degree (poly)
+  (:documentation "Get the degree of a polynomial"))
+
+(defgeneric call (poly x)
```

Comment:
Seems to me that `call` is a bit to generic name, especially on the whole `math` package level.